### PR TITLE
New abroad people filter && New tools for filters

### LINKED ISSUE #250: Abroad people filter to new system.

### LINKED ISSUE #331: Abroad item filter showing on the people page.
image

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #250.
Closes #331.

### CODE REVIEW

**Comment by Sashank999** [comment]
If the new system is okay, I will also adopt the Abroad Items Filter to this system.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
The filter should not move when the statistic label changes size.

File: extension/scripts/features/abroad-items-filter/ttAbroadItemsFilter.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 "use strict";
 
 (async () => {
-	if (!getPageStatus().access) return;
 	if (!isAbroad()) return;
+	if (!getPageStatus().access) return;
+	if (getSearchParameters().get("page")) return;
```
Comment (DeKleineKobini) [inline extension/scripts/features/abroad-items-filter/ttAbroadItemsFilter.js:6]: Should be `has`.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

File: extension/scripts/features/abroad-items-filter/ttAbroadItemsFilter.js (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/features/abroad-items-filter/ttAbroadItemsFilter.js]: Done.

**Pull request merged**